R/plotStorms.R

```diff
@@ -313,22 +321,23 @@ plotStorms <- function(sts,
   }
 
   # Plotting track(s) and labels
-  lapply(stsAux, plotTrack)
+  lapply(stsAux, plotTrack, sds)
   if (labels) {
     lapply(stsAux, plotLabels, by, pos)
   }
 
   # Adding legends
   if (legends != "none") {
-    leg <- c("TD", "TS", "Cat. 1", "Cat. 2", "Cat. 3", "Cat. 4", "Cat. 5")
-    col <- c("#00CCFF", "#00CCCC", "#FFFFB2", "#FECC5C", "#FD8D3C", "#F03B20", "#BD0026")
-
-    lty <- rep(0, 7)
-    pch <- rep(19, 7)
-    lwd <- rep(1, 7)
+    
+    leg <- names(sds@scalePalette)
```

Comment:
Could it be that by default:
- `scalePalette` is based on `mswSshsPalette` (defined in `internalData.R`) and the number of intervals in `scale`
- `names(scalePalette)` is `c("Cat. 1", "Cat. 2", ...)`

Reply:
Nop, `msmSshsPalette` is defined for `plotBehavior()` to plot raster data, `scalePalette` is used for the tracks. I think I might remove  `msmSshsPalette` in further commit and defined a more general default palette